docs/building-on-lisk/using-oracle-data/redstone-push.md

```diff
@@ -0,0 +1,215 @@
+---
+title: ...with Redstone (Push)
+slug: /building-on-lisk/using-oracle-data/redstone-push
+description: A guide on using Redstone Push to access real-world data such as asset prices, directly from your smart contracts on the Lisk blockchain.
+keywords: [
+    Oracle
+    Oracles,
+    Redstone,
+    price feeds,
+    data feeds,
+    smart contract,
+    Lisk blockchain,
+    Lisk network,
+    Lisk testnet,
+    Lisk test network,
+    app development,
+    dapp development,
+    build a dapp on Lisk,
+    build on Lisk,
+    Redstone Push,
+  ]
+---
+
+# Accessing real-world data using the Redstone Oracle (Push)
+
+This page will explain how you can access oracle data using [Redstone Push](https://docs.redstone.finance/docs/get-started/models/redstone-push).
+
+RedStone is a data ecosystem that delivers frequently updated, reliable, and diverse data for your dApp and smart contracts deployed on Lisk.
+
+RedStone data feeds are compatible with Chainlinks [AggregatorV3Interface](https://docs.chain.link/data-feeds/using-data-feeds#solidity) and include the following components:
+
+- **Aggregator contract**: An aggregator is a contract that receives periodic data updates from the oracle network.
+Aggregators store aggregated data onchain so that consumers can retrieve it and act upon it within the same transaction.
+These contracts are already deployed on the Lisk network and can be directly used by Consumers.
+- **Consumer**: A consumer is an onchain or offchain application that uses Data Feeds.
+Consumer contracts use the `AggregatorV3Interface` to call functions on the proxy contract[^1] of the Aggregator to retrieve oracle data.
+
+[^1]: Proxy contracts are onchain proxies that point to the aggregator for a particular data feed.
+Using proxies enables the underlying aggregator to be upgraded without any service interruption to consuming contracts.
+
+## Data feeds on Lisk
+The following Aggregators are available on Lisk Mainnet for Redstone Push:
+
+- [ETH/USD L2PriceFeedWithoutRounds](https://blockscout.lisk.com/address/0x6b7AB4213c77A671Fc7AEe8eB23C9961fDdaB3b2)
+  - address: `0x6b7AB4213c77A671Fc7AEe8eB23C9961fDdaB3b2`
+- [LSK/USD L2PriceFeedWithoutRounds](https://blockscout.lisk.com/address/0xa1EbA9E63ed7BA328fE0778cFD67699F05378a96)
+  - address: `0xa1EbA9E63ed7BA328fE0778cFD67699F05378a96`
+- [USDT/USD L2PriceFeedWithoutRounds](https://blockscout.lisk.com/address/0xd2176Dd57D1e200c0A8ec9e575A129b511DBD3AD)
+  - address: `0xd2176Dd57D1e200c0A8ec9e575A129b511DBD3AD`
```

Comment:
We've just recently deployed two new data feed contracts if you want to add them:
- [USDC/USD](https://blockscout.lisk.com/address/0xb4e6A7861067674AC398a26DD73A3c524C602184)
- [WBTC/USD](https://blockscout.lisk.com/address/0x13da43eA89fB692bdB6666F053FeE70aC61A53cd)